packages/ui-types/src/types/layout.tsx

```diff
@@ -2,7 +2,14 @@ import { TitleProps, LayoutProps } from "@pankod/refine-core";
 
 export type RefineLayoutHeaderProps = {};
 
-export type RefineLayoutSiderProps = {};
+export type RefineLayoutSiderProps = {
+    bottomSection?:
+        | React.ReactNode
+        | ((defaultBottom: React.ReactNode) => React.ReactNode);
+    topSection?:
```

Comment:
```suggestion
    bottom?:
        | React.ReactNode
        | ((defaultBottom: React.ReactNode) => React.ReactNode);
    top?:
```

How is it with the new names like this?